README.md

```diff
@@ -45,3 +45,4 @@ Finally, if you are sure no one else had the issue, follow the **Issue template*
 We welcome contributions to the Arduino ESP32 project!
 
 See [contributing](https://docs.espressif.com/projects/arduino-esp32/en/latest/contributing.html) in the documentation for more information on how to contribute to the project.
+TRIGGER
```

Comment:
would you please remove this so that we can merge your PR :)

Reply:
This is just a dummy commit to trigger the workflow and see if that cache was restored correctly.  I'll drop it in a second.
In my last push, when the IDE was cached I saw that the sketches were skipped and not built.  I dropped the IDE commit, I'll reapply it and see what's going on.

Reply:
I added `workflow_dispatch` to be able to trigger the workflow manually, this hack won't be needed anymore (i.e. got tired of force pushing...)